### DISCUSSION TITLE
Fuse mount speed low

### DISCUSSION THREAD

**Discussion by summerisc** [seaweedfs/seaweedfs#1920] [Q&A] [answered]
Trying to copy from local disk to mounted path, the speed is pretty slow as the picture shows:
![Screen Shot 2021-03-17 at 6 34 45 PM](https://user-images.githubusercontent.com/21230663/111547380-7335bd80-874f-11eb-8b33-a34d12832140.png)
Anything I should do or might be the issue? This directory is mounted/unmount a few times if this matters.

**Comment by chrislusf** [comment, marked as answer]
Make sure you are using version 2.34

**Reply by summerisc** [reply]
tried 2.34 and it has around 1 mb/ sec, is it a normal speed? seems pretty slow compared with the design.

**Reply by chrislusf** [reply]
can not optimize here. need to make a reproducible case.
